allennlp/data/dataset_readers/dataset_reader.py

```diff
@@ -38,20 +38,20 @@ class WorkerInfo:
 @dataclass
 class DistributedInfo:
     """
-    Contains information about the node rank and world size when the reader is being
+    Contains information about the process rank and world size when the reader is being
     used within distributed training.
 
     From a `DatasetReader` this can be accessed with the [`get_distributed_info()`](#get_distributed_info) method.
     """
 
     world_size: int
     """
-    The total number of distributed nodes.
+    The total number of processes in the distributed group.
     """
 
-    node_rank: int
+    global_rank: int
     """
-    The 0-indexed ID of the current node.
+    The 0-indexed ID of the current process within the distributed group.
```

Comment:
```suggestion
    The 0-indexed ID of the current process within the distributed group. This will be between 0 and `world_size - 1`,
    inclusive.
```